src/net.h

```diff
@@ -346,6 +346,15 @@ class CConnman
     bool RemoveAddedNode(const std::string& node);
     std::vector<AddedNodeInfo> GetAddedNodeInfo();
 
+    /**
+     * Attempts to open a connection. Currently only used from tests.
+     *
+     * @param[in]   address     Address of node to try connecting to
+     * @param[in]   conn_type   ConnectionType::OUTBOUND or ConnectionType::BLOCK_RELAY
+     * @return      bool        Returns false if at max connection capacity
```

Comment:
nit: "max outbound connection"

Reply:
? 

returns false if its at max connection of the specified type (full relay or block relay), as well as checking total outbound connections

Reply:
That was a nit, to specify _outbound_ at argument comment :p

Reply:
I agree with ariard here, I would prefer this comment was longer and more verbose.  "Returns false if at max connection capacity for given connection type or at max total outbound connection capacity"